System log on Aspire One fills with messages: fix by changing
them to debug messages.

Index: linux-2.6.27.noarch/drivers/pci/hotplug/pciehp_ctrl.c
===================================================================
--- linux-2.6.27.noarch.orig/drivers/pci/hotplug/pciehp_ctrl.c
+++ linux-2.6.27.noarch/drivers/pci/hotplug/pciehp_ctrl.c
@@ -117,13 +117,13 @@ u8 pciehp_handle_presence_change(struct 
 		/*
 		 * Card Present
 		 */
-		info("Card present on Slot(%s)\n", slot_name(p_slot));
+		dbg("Card present on Slot(%s)\n", slot_name(p_slot));
 		event_type = INT_PRESENCE_ON;
 	} else {
 		/*
 		 * Not Present
 		 */
-		info("Card not present on Slot(%s)\n", slot_name(p_slot));
+		dbg("Card not present on Slot(%s)\n", slot_name(p_slot));
 		event_type = INT_PRESENCE_OFF;
 	}
 
